common/shlibs

```diff
@@ -4000,3 +4000,7 @@ libnotcurses.so.2 notcurses-2.0.4_1
 libnotcurses++.so.2 notcurses-2.0.4_1
 libevemu.so.3 evemu-2.7.0_1
 libantilib.so.1 libantimicrox-3.1.2_1
+libwaylandpp-cursor++.so.0 libwaylandpp-0.2.8_1
+libwaylandpp-client++.so.0 libwaylandpp-0.2.8_1
+libwaylandpp-client-extra++.so.0 libwaylandpp-0.2.8_1
+libwaylandpp-egl++.so.0 libwaylandpp-0.2.8_1
```

Comment:
Why add these? We tend to not add them unless required by some package.

Reply:
I thought maybe at some point something will depend on them and atleast lets us know if there is a SONAME bump happening as well.

Reply:
Hmm, I think we only care about the SONAME bump once something actually depends on it.